lib/sundials-config/prepare.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SUNDIALS_VERSION="4.0.1"
+
+SUNDIALS_TAR=sundials-${SUNDIALS_VERSION}.tar.gz
+
+tar xvzf $SUNDIALS_TAR
+mv sundials-${SUNDIALS_VERSION} sundials_${SUNDIALS_VERSION}
+
+cd sundials_${SUNDIALS_VERSION}
+
+rm -rf INSTALL_GUIDE.pdf config/ doc/ examples/ test/ */cvode */ida */arkode */kinsol
+find . -name CMakeLists.txt -exec rm {} \;
+
```

Comment:
At this point, we should `git add sundials_${SUNDIALS_VERSION}`. This is the clean, direct addition of all the Sundials code.

Follow that by adding something like:
```
git commit -m "Upgrading Sundials to version ${SUNDIALS_VERSION}"
```